project.Makefile

```diff
@@ -1 +1,5 @@
-## Add your own custom Makefile targets here
+CLI=$(RUN) python -m src.comp_loinc
```

Comment:
## `makefile` setup
At first I was going to delete `project.Makefile`. I thought maybe this was poetry related, but I googled around and it seemed like it wasn't and that we didn't need it. Then I realized that the `makefile` in this project is LinkML cookie cutter. When I worked on this project last, this wasn't the case.

I'm kind of annoyed that `all` has been taken by the cookie cutter, because that is pretty standard and was what I was going to use.

I really don't know much about LinkML cookiecutter; guess I'll also be learning.